user/mods/ServerValueModifier/Presets/The Escape.json

```diff
@@ -116,14 +116,14 @@
       "Therapist": 1
     },
     "MinDurabSell": 40,
-    "RemoveTimeCondition": false,
+    "RemoveTimeCondition": true,
     "AllQuestsAvailable": false,
     "RemoveBarterOffers": false,
     "RemoveCurrencyOffers": false,
     "IncreaseAssort": false,
     "UnlockQuestAssort": false,
     "EnableTraders": true,
-    "FIRRestrictsQuests": false,
+    "FIRRestrictsQuests": true,

```

Comment:
This is already handled by Realism

Reply:
From my experience, it's better to keep it enabled even with realism